lib/ace/mode/javascript/jshint.js

```diff
@@ -2392,13 +2392,13 @@ module.exports = slice;
     exports.noConflict = function () { global._ = current; return exports; };
   }()));
 }(this, (function () {
-  //     Underscore.js 1.13.4
+  //     Underscore.js 1.13.6
   //     https://underscorejs.org
   //     (c) 2009-2022 Jeremy Ashkenas, Julian Gonggrijp, and DocumentCloud and Investigative Reporters & Editors
   //     Underscore may be freely distributed under the MIT license.
 
   // Current version.
-  var VERSION = '1.13.4';
+  var VERSION = '1.13.6';
```

Comment:
In the commit title it says `2.13.6` but here it says `1.13.6`, is that correct?

Reply:
It's just interesting coincidence. It's version of updated `underscore` package

Reply:
ah I see, they are different packages all together that just coincidentally are on very similar version numbers, thx